apps/web/src/pages/About/index.tsx

```diff
@@ -0,0 +1,15 @@
+import Members from '@/components/Members';
+import React from 'react';
+import { StyledAbout } from './styles';
+
+const About = () => {
+  return (
+    <>
```

Comment:
not necessary, please remove.

Reply:
`const About = () => (
  <StyledAbout>
    <Members />
  </StyledAbout>
);`

i changed the file to this.